GTMAppAuthSwift/Tests/ObjCIntegration/GTMKeychainStoreTests.m

```diff
@@ -92,7 +92,7 @@ - (void)testSavingWithNewItemName {
   XCTAssertNil(error);
```

Comment:
It would be good to verify the class name that `NSKeyedArchiver` is encoding at this point.

Reply:
I've added a handful of tests. Two to document the backwards and forwards compatibility checking the `NSKeyed(Un)Archiver`'s class name mapping. Two to inspect the data produced by the archivers to verify that the data has the right class name.